rai/node/node.cpp

```diff
@@ -2176,27 +2176,27 @@ class distributed_work : public std::enable_shared_from_this<distributed_work>
 											}
 											else
 											{
-												BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Work peer %1% responded with an error %2%") % connection->address % connection->port);
+												BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Work peer responded with an error %1% %2%: %3%") % connection->address % connection->port % connection->response.result ());
 												this_l->failure (connection->address);
 											}
 										}
 										else
 										{
-											BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to read from work_peer %1% %2%") % connection->address % connection->port);
+											BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to read from work_peer %1% %2%: %3%") % connection->address % connection->port % ec);
 											this_l->failure (connection->address);
 										}
 									});
 								}
 								else
 								{
-									BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to write to work_peer %1% %2%") % connection->address % connection->port);
+									BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to write to work_peer %1% %2%: %3%") % connection->address % connection->port % ec);
```

Comment:
Do we want to see the code or ec.message()?

Reply:
Good question. Not sure if a message is always present, so maybe both? Error message with code in parenthesis sound good?

Reply:
Oh yea, that's a good idea.

Reply:
Updated